samples/PT37_opengpu/yolov5s_pt37_app/packages/028663699634-yolov5s_pt37_app-1.0/src/app/app_fp16.py

```diff
@@ -100,13 +107,18 @@ def run(self):
                 # e.g. with batch size 4, have [4, 1, 3, 640, 640] squeezed into [4, 3, 640, 640]
                 pre_processed_images = torch.squeeze(pre_processed_images, dim=1)
 
+                # PyTorch CUDA is asynchronous.
+                # For time measurement, call synchronize() to wait for the prior GPU operation's completion.
```

Comment:
Maybe you can mention that this can be disabled if we don't need to do time measurement.
I guess using sync explicitly will slightly impact the overall performance.